src/OVAL/probes/probe/icache.c

```diff
@@ -334,16 +345,22 @@ probe_icache_t *probe_icache_new(void)
 
 static int __probe_icache_add_nolock(probe_icache_t *cache, SEXP_t *cobj, SEXP_t *item, pthread_cond_t *cond)
 {
-        assume_d((cond == NULL) ^ (item == NULL), -1);
```

Comment:
`!((cond == NULL) ^ (item == NULL))` can be rewritten to `(cond == NULL && item == NULL) || (cond != NULL && item != NULL)`

Reply:
OK